app/seaf-cli

```diff
@@ -583,6 +583,7 @@ def seaf_sync(args):
     repo_name = tmp['repo_name']
     version = tmp.get('repo_version', 0)
     repo_salt =  tmp['salt']
+    is_readonly = tmp['permission']
```

Comment:
When connecting to an old version, permission is not in the response. You need to handle this case.